Files/Strings/en-US/Resources.resw

```diff
@@ -984,6 +984,11 @@
   <data name="ConfirmDeleteDialogDeleteOneItem.Text" xml:space="preserve">
     <value>Are you sure you want to delete this item?</value>
   </data>
+  <data name="VerticalTabViewAddTab.Text" xml:space="preserve">
+    <value>Add tab</value>
+  </data>
+  <data name="VerticalTabViewHeader.Text" xml:space="preserve">
+    <value>Open tabs</value>
```

Comment:
```suggestion
    <value>Open tabs</value>
</data>
```

Reply:
@yaichenbaum I actually used the MAT extension but I needed to resolve merge conflicts